org.freecad.FreeCAD.yaml

```diff
@@ -107,24 +106,27 @@ modules:
     post-install:
       # Needed for graphviz
       - ln -s tclsh8.6 /app/bin/tclsh
+      - chmod -R u+w ${FLATPAK_DEST}/lib/libtcl*
     sources:
       - type: archive
-        url: https://downloads.sourceforge.net/project/tcl/Tcl/8.6.8/tcl8.6.8-src.tar.gz
-        sha256: c43cb0c1518ce42b00e7c8f6eaddd5195c53a98f94adc717234a65cbcfd3f96a
+        url: https://downloads.sourceforge.net/project/tcl/Tcl/8.6.16/tcl8.6.16-src.tar.gz
+        sha256: 91cb8fa61771c63c262efb553059b7c7ad6757afa5857af6265e4b0bdc2a14a5
```

Comment:
do you know why we have tcl and tk here?

Reply:
it's a dependency for graphviz if I remember.

Upgrade is necessary for the newer compilers.